.github/workflows/coverage.yaml

```diff
@@ -76,14 +105,16 @@ jobs:
       - name: Parse converted tests from converted-ethereum-tests.txt
         run: |
           echo "New lines introduced in converted-ethereum-tests.txt:"
-          lines=$(git diff origin/${{ github.base_ref }} HEAD -- converted-ethereum-tests.txt | grep "^+" | grep -v "^+++")
-          files=$(echo "$lines" | grep -oP '(?<=\+).+\.json')
-
-          if [ -z "$files" ]; then
-              echo "Error: No new JSON files found in converted-ethereum-tests.txt"
-              exit 1
+          lines=$(git diff origin/${{ github.base_ref }} HEAD -- converted-ethereum-tests.txt | grep "^+" | grep -v "^+++" || true)
```

Comment:
It'd be nice to simplify this. As this file is so central to producing the coverage diff, I think we should replace this plain-text with either:
1. A CSV, YAML,... file, or,
2. Build this functionality directly into `fill`, so that calling something along of `fill --show-ported` would generate structured output that could then be more readily parsed by this workflow. 

I favor 2., as it would:
1. Avoid diffing to pull a list of relevant files.
2. Allow us to even map parametrized test cases to ethereum/tests json (so we can potentially even more finely tune the diff). For example:
    ```python
          pytest.param(
          Op.CREATE2,
          marks=pytest.mark.ports(["tests/BlockchainTests/GeneralStateTests/Cancun/test.json"),
          id="create2",
      ),
    ```

Happy to discuss!

Reply:
how would you get the files that are needed to verify on a PR?

Reply:
I would add a supplemental pytest plugin to fill that simply writes a list of files (.json from ethereum/tests) that this test function claims to have ported (and skips executing the tests). Either to the specified file or to stdout, prob file because pytest is noisy (and often has helpful noise).

I.e.,
1. Get list of python files changed in `./tests`.
2. Determine all tests effected by these changes. As finely as possible, but we'll most likely have to operate on a test module level, unfortunately.
3. Ask `fill -p src.pytest_plugins.filler.ported_json` if and which json these tests (modules) ported.
4. Generate fixtures using `fill`.
5. If ported, generate fixtures using `retesteth`.

Reply:
for future prs